.eslintrc

```diff
@@ -0,0 +1,135 @@
+{
+  "extends": ["airbnb", "prettier", "prettier/react"],
```

Comment:
airbnb? :)

Reply:
Yaa Airbnb developers have created the golden standard of linting. Its a very common linting setup to use.